account_advance_payment/__openerp__.py

```diff
@@ -25,7 +25,7 @@
 #
 {
     "name": "Account Advance Payment",
-    "version": "1.6",
+    "version": "9.0.0.1.6",
```

Comment:
The true of `1.6` is that is a copy&paste of other module.
Don't represent a real version.
It just was added as reference.